EMS/employeedata.json

```diff
@@ -0,0 +1,189 @@
+[
+  {
+    "EmployeeNumber": "TZ102",
+    "FirstName": "rohan",
+    "LastName": "karthik",
+    "Dob": "15/04/2003",
+    "EmailId": "rohan@gmail.com",
+    "MobileNumber": 1234567891,
+    "JoiningDate": "15/01/2024",
+    "Location": "Mumbai",
+    "LocationID": 1,
+    "JobTitle": "UIUXDesigner",
```

Comment:
Json that is being stored doesn't need to have Name fields for enums. It should have only ids.